### ISSUE TITLE
Add keyboard interaction for DropDownMenu

### ISSUE THREAD

**Issue by ramtinabadi** [Vieolo/vieolo-ui#39]
The component should be focusable using tab, opened using space or enter, and user should be able to select options using arrow keys and enter
